SAKPaaS/src/app/components/footer/footer.component.html

```diff
@@ -1,20 +1,53 @@
 <div class="footer">
-    <a target="_blank" rel="noreferrer" href="https://www.instagram.com/happyhamsterorg/">
-        <img src="assets/icons/social-media/instagram.svg" alt="Instagram">
-    </a>
-    <a target="_blank" rel="noreferrer" href="https://twitter.com/happyhamsterorg">
-        <img src="assets/icons/social-media/twitter.svg" alt="Twitter">
-    </a>
-    <a target="_blank" rel="noreferrer" href="https://github.com/SAKPaaS">
-        <img src="assets/icons/social-media/github.svg" alt="GitHub">
-    </a>
-    <a target="_blank" rel="noreferrer" href="https://www.youtube.com/channel/UCEYDgvcxXbOLIrKXZcC4nCA">
-        <img src="assets/icons/social-media/youtube.svg" alt="Youtube">
-    </a>
-    <a [routerLink]="" (click)="toggleLanguage()">
-        {{ selectedLanguage === 'de' ? 'EN' : 'DE'}}
-    </a>
-    <div class="line-break"></div>
-    <a href="https://welcome.happyhamster.org/">{{ 'footer.help' | translate }}</a>
-    <a href="" routerLink="/legal">{{ 'footer.legal' | translate }}</a>
+  <a
+    target="_blank"
+    rel="noreferrer"
+    href="https://www.instagram.com/happyhamsterorg/"
+  >
+    <img src="assets/icons/social-media/instagram.svg" alt="Instagram" />
+  </a>
+  <a
+    target="_blank"
+    rel="noreferrer"
+    href="https://twitter.com/happyhamsterorg"
+  >
+    <img src="assets/icons/social-media/twitter.svg" alt="Twitter" />
+  </a>
+  <a target="_blank" rel="noreferrer" href="https://github.com/SAKPaaS">
+    <img src="assets/icons/social-media/github.svg" alt="GitHub" />
+  </a>
+  <a
+    target="_blank"
+    rel="noreferrer"
+    href="https://www.youtube.com/channel/UCEYDgvcxXbOLIrKXZcC4nCA"
+  >
+    <img src="assets/icons/social-media/youtube.svg" alt="Youtube" />
+  </a>
+  <a
+    title="{{ 'footer.title.feedback' | translate }}"
+    target="_blank"
+    rel="noreferrer"
+    href="https://docs.google.com/forms/d/e/1FAIpQLSekXXSN--ehheZoMnMj9oE3tzCdS0C8oQ0lg8Lpl8RxNRIPEA/viewform"
+  >
+    <svg
```

Comment:
warum kopierst du direkt das svg hier ein?
du kannst auch direkt `<mat-icon>` verwenden.

Reply:
Tell me how. `<mat-icon>feedback outline</mat-icon>` didn't seem to work.

Reply:
Das ist 💩 dokumentiert.
`<mat-icon class="material-icons-outlined">feedback</mat-icon>` sollte funktionieren.